frontend/packages/knative-plugin/src/components/routes/RouteDetailsPage.tsx

```diff
@@ -1,26 +1,15 @@
 import * as React from 'react';
 import { Kebab, navFactory } from '@console/internal/components/utils';
 import { DetailsPage } from '@console/internal/components/factory';
-
 import { DetailsForKind } from '@console/internal/components/default-resource';
-import { useServingBreadcrumbsFor } from '../services/hooks';
 import { RouteModel } from '../../models';
 
 const RouteDetailsPage: React.FC<React.ComponentProps<typeof DetailsPage>> = (props) => {
-  const { kindObj, match } = props;
-  const breadcrumbsFor = useServingBreadcrumbsFor(kindObj, match);
```

Comment:
Reverted this change for breadcrumbs to point to the correct list page and be consistent with other details page.

The current console convention is that the breadcrumb generally serves the purpose of conveying the hierarchy of a list view to an individual resource, and not necessarily the navigation order.

http://openshift.github.io/openshift-origin-design/conventions/documentation/interactions.html#breadcrumbs